script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -987,7 +991,8 @@ One after another.
 [sha:d9a0d8f5bb438a257fd746351cf95f4f141f3601]{
 -- Page 633, Offset 35093.
 -- 二人目とか三人目とか、生きたままで箱になったりカーペットになったり、めまぐるしい。
-Seeing people like the second and third victim turning into things like boxes and carpets was enough to make me dizzy.
+// how does this work???
```

Comment:
maybe this is a japanese expression that someone more fluent than i would understand, but my best guess is that theyre being smooshed flat or smooshed in